apps/andi/lib/andi/input_schemas/datasets.ex

```diff
@@ -111,7 +113,7 @@ defmodule Andi.InputSchemas.Datasets do
       |> StructTools.to_map()
       |> Map.merge(form_changes)
 
-    new_changes = %{technical: technical_changes, business: business_changes, id: dataset_id} |> StructTools.to_map()
+    new_changes = %{technical: technical_changes, business: business_changes, id: dataset_id}
```

Comment:
Why no to_map anymore?

Reply:
They are all maps already, this was doing nothing